website/static/install.sh

```diff
@@ -38,6 +40,8 @@ while getopts ":hd:t:" option; do
          install_dir=${OPTARG};;
```

Comment:
To include the `-v` option for specifying a version, you should modify the `getopts` string. Here's the change:

```
while getopts ":hd:t:v:" option; do
```

This will allow the script to handle the `-v` option correctly, requiring an argument for the version.